src/hardware/lpt_dac.cpp

```diff
@@ -197,4 +220,7 @@ void LPT_DAC_Init(Section* section)
 
 	constexpr auto changeable_at_runtime = true;
 	section->AddDestroyFunction(&LPT_DAC_ShutDown, changeable_at_runtime);
+
+	lpt_dac->output_queue.Resize(iceil(lpt_dac->channel->GetFramesPerBlock() * 2.0f));
```

Comment:
What's the 2.0f?

Reply:
No, it's just twice the blocksize so we have an extra bit of buffering.  It's frames per block, not samples.

Reply:
Can you put an explanatory comment above all the occurrences then? It's not immediately obvious why oversizing by a factor of 2 is needed and the guy might be tempted to "correct" it and remove the `* 2.0f` 😄

Reply:
Commented.